lib/web_ui/lib/src/ui/channel_buffers.dart

```diff
@@ -0,0 +1,33 @@
+// Copyright 2013 The Flutter Authors. All rights reserved.
+// Use of this source code is governed by a BSD-style license that can be
+// found in the LICENSE file.
+
+part of ui;
+
+typedef DrainChannelCallback = Future<void> Function(ByteData, PlatformMessageResponseCallback);
```

Comment:
dartdoc

Reply:
I'm not sure what you are referring to.  I updated the docstring to match the ones found in `lib/ui/window.dart`